### DISCUSSION TITLE
Retrieve my lost github account

### DISCUSSION THREAD

**Discussion by Prajwalg19** [community/community#143214] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Somebody hacked my gmail and got access to my github. They have removed all the ssh keys from the account and enabled TOTP based 2 factor authentication. Is there by any means i can get back my lost github account because currently even if pass the email and password authentication i will be stuck at the TOTP step which i don't have access to login.

**Comment by LiteBrite82** [comment]
Hi @Prajwalg19 ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
